clang/lib/CIR/CodeGen/CIRGenTypeCache.h

```diff
@@ -35,9 +35,10 @@ struct CIRGenTypeCache {
   // usigned char, unsigned, unsigned short, unsigned long
   mlir::cir::IntType UInt8Ty, UInt16Ty, UInt32Ty, UInt64Ty;
   /// half, bfloat, float, double
-  // mlir::Type HalfTy, BFloatTy;
   // TODO(cir): perhaps we should abstract long double variations into a custom
   // cir.long_double type. Said type would also hold the semantics for lowering.
```

Comment:
Looks like this comment isn't relevant anymore? You might as well delete it already

Reply:
Updated.